src/docs/operator-guides/how-to-create-an-faq-page/index.md

```diff
@@ -0,0 +1,295 @@
+
+Let’s use a custom appearance, with a different background color from the other Sections (RGB 90,90,90) and white text. Then, we can add as many Blocks as we want the users to be able to browse through, in this case, 4. Each of these Blocks can get a fitting ID (such as review-1), a quote from the review in the title, and the full review in the ‘Block text’ Field.
+
+This is the last Section we’ll create: the footer of the marketplace automatically appears at the very bottom of the page.
+
+Reference images: 
+
+<extrainfo title="Review Section configuration">
+
+![Change environments](./faq-5-13-faq-section-3.jpg)
+
+</extrainfo>
+
+<extrainfo title="Block 1 configuration">
+
+![Change environments](./faq-5-14-review-block-1.jpg)
+
+</extrainfo>
+
+<extrainfo title="Block 2 configuration">
+
+![Change environments](./faq-5-15-review-block-2.jpg)
+
+</extrainfo>
+
+<extrainfo title="Block 3 configuration">
+
+![Change environments](./faq-5-16-review-block-3.jpg)
+
+</extrainfo>
+
+
+
+## Dealing with errors
+
+Sometimes when trying to save the page, you may run into an error. If the page is not fully functional and you try to save the changes, a red outline is created around the section(s) that need fixing, and the changes are not saved. For example, if you create a new Section and otherwise fill it out properly but do not give it a Section ID, you won’t be able to save the page before giving a name to that Section, because every Section needs an ID.
+
+If, when you are saving your changes, you don’t see a checkmark and the “Save changes” button does not become grayed out, it means that the changes were not saved. In this case, scroll up to the Section(s) outlined in red, and open them. You will find the error message typed out under any field that needs to be fixed.
+
+Reference images:
+
+<extrainfo title="Error Section outlined">
+
+![Change environments](./faq-6-1-errors-1.jpg)
+
+</extrainfo>
+
+<extrainfo title="error labeled within Section">
+
+![Change environments](./faq-6-2-errors-2.jpg)
+
+</extrainfo>
+
+
+## Setting the page live
+
+Now that we’ve created the page and made sure it works as supposed, all that’s left is to move it to the live environment - in other words, to production. This can be done by clicking the blue button “Copy to production…” on the left side bar. 
+
+When you click on the button, you’ll get a pop-up asking you to confirm that you wish to move all the changes to production. You don’t need to move all of the changes at the same time, though: if you have made multiple changes to different pages, you can pick and choose which ones you want to move to production. 
+
+A word of warning here, as well: as mentioned in the pop-up, the old version(s) of the page(s) in production will be overwritten and this action cannot be undone.
```

Comment:
"A word of warning here, as well: if you've made changes directly to your production environment, those changs will be overwritten and this action cannot be undone."